docs/usage.md

```diff
@@ -288,16 +288,23 @@ spec:
 If you set `replicas` to more than 1, normally you should not set `sessionAffinity` to `None`.
 This is because session affinity is mandatory to keep stateful TCP connections.
 
-One problem with session affinity in Kubernetes is that it limits the maximum
-duration of a session.  By default, [it is 3 hours](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#clientipconfig-v1-core).
+You can extend the timeout duration for idle connections with 
```

Comment:
Possibly this line is a duplication of the following line?

Reply:
oops.  fixed.